spoon-runner/src/main/java/com/squareup/spoon/SpoonRunner.java

```diff
@@ -99,6 +101,91 @@ private SpoonRunner(String title, File androidSdk, File applicationApk, File ins
     }
   }
 
+  /** Returns {@code false} if a test failed on any device. */
+  static boolean parseOverallSuccess(SpoonSummary summary) {
```

Comment:
apologies I will commit the requested changes thanks for the review

most probably I have the wrong version of codestyle(?) I used: https://github.com/square/java-code-styles

my IDE:
IntelliJ IDEA 2016.1.1 Community Edition
Build #IC-145.597, built on March 29, 2016

Both SquareAndroid and Square codestyles seem to reformat it:
Optimize imports & Rearrange code  & Auto Indent lines
But then again if 2 people confirmed it it is probably just a config issue somewhere on my side then.
I will commit required code style trough the next hour cheers :+1: